### ISSUE TITLE
Error with llvmlite

### ISSUE THREAD

**Issue by aleksejs-fomins** [johannfaouzi/pyts#87]
`pip install pyts`

`Failed building wheel for llvmlite`

`RuntimeError: llvm-config failed executing, please point LLVM_CONFIG to the path for llvm-config`

**Comment by johannfaouzi** [comment]
Could you give more information about your system? OS, python version, etc. It's more a `llvmlite` issue than a `pyts` issue.

**Comment by aleksejs-fomins** [comment]
Yes, it seems to have been a pre-requisite for numba, which could not be installed. When I tried installing numba through conda-forge, it worked fine. I'm on Ubuntu 18.04 (4.15.0-122-generic), Python 3.6
